docs/samples/browser-plugin-meetings/app.js

```diff
@@ -3334,7 +3339,9 @@ async function toggleBrb() {
   const meeting = getCurrentMeeting();
 
   if (meeting) {
-    const enabled = document.getElementById('brb').checked;
+    const brbButton = document.getElementById('brb-btn');
+    const enabled = brbButton.innerText !== 'Remove away';
+
```

Comment:
What about the button state, should it changed after brb is applied? 
After I pressed Apply away the state (name) is still the same (Apply away).

Reply:
With two users I have such behavior, what test case did you have to check it?

<img width="1176" alt="Screenshot 2024-12-13 at 14 19 35" src="https://github.com/user-attachments/assets/f7d92e24-a875-4e4e-a36a-3660ea88fa8c" />
<img width="1210" alt="Screenshot 2024-12-13 at 14 19 30" src="https://github.com/user-attachments/assets/9655d0ad-c7ea-4b47-8c6e-1b8abbfc6896" />